specification/cognitiveservices/data-plane/Language/stable/2022-05-01/conversationalanalysisauthoring.json

```diff
@@ -0,0 +1,3060 @@
+            "$ref": "common.json#/parameters/ProjectNamePathParameter"
+          },
+          {
+            "in": "path",
+            "name": "trainedModelLabel",
+            "description": "The trained model name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Trained model info",
+            "schema": {
+              "$ref": "#/definitions/ConversationalAnalysisAuthoringProjectTrainedModel"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Get Model": {
+            "$ref": "./examples/conversationalanalysisauthoring/SuccessfulGetModel.json"
+          }
+        }
+      },
+      "delete": {
+        "description": "Deletes an existing trained model.",
+        "operationId": "ConversationalAnalysisAuthoring_DeleteTrainedModel",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ProjectNamePathParameter"
+          },
+          {
+            "in": "path",
+            "name": "trainedModelLabel",
+            "description": "The trained model name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Deleted successfully."
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Delete Model": {
+            "$ref": "./examples/conversationalanalysisauthoring/SuccessfulDeleteModel.json"
+          }
+        }
+      }
+    },
+    "/authoring/analyze-conversations/projects/{projectName}/models/{trainedModelLabel}/evaluation/result": {
+      "get": {
+        "description": "Gets the detailed results of the evaluation for a trained model. This includes the raw inference results for the data included in the evaluation process.",
+        "operationId": "ConversationalAnalysisAuthoring_GetModelEvaluationResults",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ProjectNamePathParameter"
+          },
+          {
+            "in": "path",
+            "name": "trainedModelLabel",
+            "description": "The trained model name.",
+            "required": true,
+            "type": "string"
+          },
```

Comment:
Nit: could also define a shared parameter for this as well. Used in the endpoint above.